ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/review/v1/review/_cq_dialog/.content.xml

```diff
@@ -0,0 +1,124 @@
+                        jcr:primaryType="nt:unstructured"
+                        sling:resourceType="granite/ui/components/coral/foundation/form/textfield"
+                        fieldLabel="Name"
+                        name="./name"
+                        required="{Boolean}true">
+                        <granite:data
+                          jcr:primaryType="nt:unstructured"
+                          node-name-validation="" />
+                      </name>
+                      <title
+                        jcr:primaryType="nt:unstructured"
+                        sling:resourceType="granite/ui/components/coral/foundation/form/textfield"
+                        fieldLabel="Title"
+                        name="./jcr:title" />
+                      <hideTitle
+                        jcr:primaryType="nt:unstructured"
+                        sling:resourceType="granite/ui/components/coral/foundation/form/checkbox"
+                        name="./hideTitle"
+                        text="Hide title"
+                        uncheckedValue="false"
+                        value="true" />
+                      <editAction
+                        jcr:primaryType="nt:unstructured"
+                        sling:orderBefore="customText"
+                        sling:resourceType="granite/ui/components/coral/foundation/form/select"
+                        granite:class="cmp-adaptiveform-review__editaction"
+                        fieldLabel="Enable edit actions for"
+                        name="./editAction"
+                        value="field">
+                        <items jcr:primaryType="nt:unstructured">
+                          <field
+                            jcr:primaryType="nt:unstructured"
+                            text="Field"
+                            value="field" />
+                          <panel
+                            jcr:primaryType="nt:unstructured"
+                            text="Panel"
+                            value="panel" />
+                          <both
+                            jcr:primaryType="nt:unstructured"
+                            text="Panel and Field"
+                            value="both" />
+                          <none
+                            jcr:primaryType="nt:unstructured"
+                            text="None"
+                            value="none" />
+                        </items>
+                      </editAction>
+                      <linkedPanels
+                        jcr:primaryType="nt:unstructured"
+                        granite:class="cmp-adaptiveform-review__linkedPanels"
+                        sling:resourceType="granite/ui/components/coral/foundation/form/select"
+                        emptyText="Select"
+                        fieldLabel="Link panels"
+                        multiple="true"
+                        value="string[]"
+                        name="./linkedPanels"
```

Comment:
We should even prefix the JCR property with fd:, all OOTB product properties should have fd: to differentiate them with customer specific properties

Reply:
Changed